cypress-tests/cypress/support/commands.js

```diff
@@ -159,6 +159,24 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.Commands.add("merchantRetrieveCallTest", (globalState) => {
+  const merchantId = globalState.get("merchantId");
+  cy.request({
+    method: "GET",
+    url: `${globalState.get("baseUrl")}/accounts/${merchantId}`,
+    headers: {
+      "Content-Type": "application/json",
+      "api-key": globalState.get("adminApiKey"),
+    },
+    failOnStatusCode: false,
+  }).then((response) => {
+    logRequestId(response.headers["x-request-id"]);
+    expect(response.headers["content-type"]).to.include("application/json");
+    expect(response.body.merchant_id).to.equal(merchantId);
+    globalState.set("publishableKey", response.body.publishable_key);
+  });
+});
+
```

Comment:
https://github.com/juspay/hyperswitch/commit/7e545e36ebdcfe46a1fcb29687af542e34d00e93#diff-b13b92cbdc1f629acd0f5406dbb85d103447168eedcbde7b750f34421b8c8f08

Merchant Retrieve Call was added by me with commit https://github.com/juspay/hyperswitch/commit/7e545e36ebdcfe46a1fcb29687af542e34d00e93

Please remove this and use that instead.